tools/pot/openvino/tools/pot/benchmark/infer_request_wrap.py

```diff
@@ -22,7 +22,7 @@ def callback(self, statusCode, userdata):
         self.callback_queue(self.id, self.request.latency)
 
     def start_async(self, input_data):
-        self.request.async_infer(input_data)
+        self.request.start_async(input_data)
 
     def infer(self, input_data):
```

Comment:
Can the async infer queue be reused from python api?

@jiwaszki

Reply:
Is this already introduced, validated, and used by other components? I heard that this feature is delayed that is why we decided to go via legacy way. But we will move to it in the future.

Reply:
@ilya-lavrenov @AlexKoff88 Actually it is introduced in https://github.com/openvinotoolkit/openvino/pull/8417 and I am awaiting reviews. It is already used by benchmark_app (pure py-implementation).